src/TTabBar.cpp

```diff
@@ -208,3 +181,24 @@ void TTabBar::applyPrefixToDisplayedText(int index, const QString& prefix)
         QTabBar::setTabText(index, QStringLiteral("%1%2").arg(prefix, tabData(index).toString()));
     }
 }
+
+void TTabBar::paintEvent(QPaintEvent* event)
+{
+    Q_UNUSED(event);
+    QStylePainter painter(this);
+    QStyleOptionTab opt;
+
+    for (int i = 0; i < count(); i++)
+    {
```

Comment:
There is a Qt standard refactor that it will offer for this construct which makes sense - to only call `count()` once - and also the standard recommendation to pre rather than post increment a variable where possible (a half decent compiler will probably do it auto-magically these days!) - and finally, for our coding style, the opening brace should be on the previous line:
```suggestion
    for (int i = 0, total = count(); i < total; ++i) {
```
🤓